frameit/spec/device_spec.rb

```diff
@@ -34,8 +34,8 @@ def expect_forced_screen_size(value)
       end
 
       it "should detect iPhone 11 Pro in portrait and landscape based on priority" do
```

Comment:
@joshdholtz why are we changing the device used in the test? 🤔 It seems like the dimensions in the test should also be changed, if we we're changing the device. Also, the spec name needs an update:

```suggestion
      it "should detect iPhone 12 Mini in portrait and landscape based on priority" do
```

But first and foremost I wanted to understand the change 😁

Reply:
The screenshot dimensions it finds are always based upon the last edit devices. So in this case the iPhone 12 mini has rather same resolution so that is the highest priority device now 🤷‍♂️ But good catch on this change!